### PR TITLE
refactor: Carousel - Move from rmrk folder

### LINKED ISSUE #3773: Refactor: Carousel - Move from `rmrk` folder
move from `components/rmrk/` folder to `components/carousel/` or `components/landing/`?
- [ ] https://github.com/kodadot/nft-gallery/blob/main/components/rmrk/Gallery/LatestSales.vue
- [ ] https://github.com/kodadot/nft-gallery/blob/main/components/rmrk/Gallery/NewestList.vue
- [ ] https://github.com/kodadot/nft-gallery/blob/main/components/rmrk/Gallery/PopularCollections.vue
- [ ] https://github.com/kodadot/nft-gallery/blob/main/components/rmrk/Gallery/GalleryItemCarousel.vue

### PR DESCRIPTION
**Pull request by preschian**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).
👇 \_ Let's make a quick check before the contribution.

### PR type

- [ ] Bugfix
- [ ] Feature
- [x] Refactoring

### What's new?

- [x] PR closes #3773  

### Before submitting Pull Request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged the recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [x] I've posted a screenshot of demonstrated change in this PR

### Optional

- [ ] I've tested it at </rmrk/collection/26902bc2f7c20c546a-1FVG7>
- [ ] I've tested PR on mobile and everything seems works
- [ ] I found edge cases
- [ ] I've written some unit tests 🧪

### Had issue bounty label?

- [x] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/transfer/?target=DY4SQF2iD456tH89aQtz5wv1EV3BbSW8wKKuMcwbmXaj1pM)

### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

### Screenshot

- [ ] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

### CODE REVIEW

**Comment by preschian** [comment]
is commented `passionFeed` safe to remove?

this PR only groups carousel-related components into the same folder. will open separate PR for migration to compose API

**Comment by yangwao** [comment]
> is commented `passionFeed` safe to remove?

actually we would like to reintroduce it again hmm 😅

**Comment by preschian** [comment]
> actually we would like to reintroduce it again hmm 😅

added back that again 👍

**Review by prachi00 — APPROVED** [review]

**Review by roiLeo — APPROVED** [review]

**Comment by yangwao** [comment]
Tests seem failed 🤔
Let's try update branch

**Review by Jarsen136 — APPROVED** [review]

File: tests/cypress/e2e/components/media.cy.ts [hunk 1]
```diff
@@ -67,15 +67,15 @@ const mediaType = [
...
-  },
+  // {
+  //   url: '/bsx/gallery/2444376227-3',
+  //   title: '3D Ball',
+  //   description: 'Smash zone adverse',
+  //   collection: 'Ping Pong Ball',
+  //   creator: 'bXhWeV...2sDcbW',
+  //   tagRelated: 'model-viewer',
+  //   type: '3d',
+  // },
```
Comment (roiLeo) [inline tests/cypress/e2e/components/media.cy.ts:78]: ☹️ is it breaking?

Comment (preschian) [reply tests/cypress/e2e/components/media.cy.ts:78]: seems this PR https://github.com/kodadot/nft-gallery/pull/3839 sometimes not really solved the issue, run e2e in local seems fine with that enabled

Comment (preschian) [reply tests/cypress/e2e/components/media.cy.ts]: oh, the test was correct, broken on bsx. checking

Comment (preschian) [reply tests/cypress/e2e/components/media.cy.ts]: sorry, turns out not working on bsx after update from main branch. will check later, currently afk 🙏🏻

**Comment by yangwao** [comment]
pay 100

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $100 @ 45.54 USD/KSM ~ 2.196 $KSM
🧗 [DY4SQF2iD456tH89aQtz5wv1EV3BbSW8wKKuMcwbmXaj1pM](https://kusama.subscan.io/account/DY4SQF2iD456tH89aQtz5wv1EV3BbSW8wKKuMcwbmXaj1pM)
🔗 [0x5ece7e7267d0f79e24e71953950b52d03ae8b944dbe678c54abc10bf02bd1b9c](https://kusama.subscan.io/extrinsic/0x5ece7e7267d0f79e24e71953950b52d03ae8b944dbe678c54abc10bf02bd1b9c)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**